### DISCUSSION TITLE
Definition des Patientennamens

### DISCUSSION THREAD

**Discussion by marcusthierfelder** [Digitaler-Impfnachweis/certification-apis#201] [Q&A]
Hallo

wir haben die **fn-Propertie** ("Family name") im Zertifikat immer als Nachnamen interpretiert. Jetzt beschweren sich Patienten mit "spezielleren" Namen darüber, das dies zu recht nicht ihr Name ist. Konkret heisst das (als PVS Anbieter halte ich mich mal an die KBV Nomenklatur) das wir 5 Namesfelder haben Vorname, Nachname, Vorsatzwort, Namenszusatz, Titel. Welche der 4 letzteren Felder müssen für **fn** genutzt werden? 

Konkretes Bsp der KBV:  Prof. Dr. Karl-Friederich Graf Freiherr von Schaumberg
was soll da in **fn**:
a) Schaumberg
b) von Schaumberg
c) Graf Freiherr von Schaumberg
d) Prof. Dr. Graf Freiherr von Schaumberg
e) Prof. Dr. von Schaumberg
...

**Comment by mge-medisoftware** [comment]
Dazu habe ich auch nie eine offizielle Information gefunden. Prinzipiell kann man wohl nur die Felder Nachname und Vorname mit Daten aus eGK befüllen und Praxen müssten dann evtl. abgleichen mit Angaben auf Personalausweis oder Reisepass. Das kommt natürlich nicht gut an... Wobei die maximale Feldlänge von jeweils 50 Zeichen hierfür theoretisch nicht ausreicht (siehe Verordnung über Personalausweise...). Es steht aber auch nirgends, dass beim Impfnachweis die Namensangaben vom Personalausweis oder Reisepass verwendet werden müssen. Obwohl je nach EU-Land wohl recht pingelig gegen Ausweis geprüft wird. Oder kennt wer ein Quelle hierzu?

In der Verordnung zum Personalausweis heißt es weiter:
In den Datenfeldern „Name“ (Familienname und Geburtsname) sowie „Vornamen“ sind alle(!?) Namensbestandteile komplett darzustellen, soweit dies technisch ... umsetzbar ist. (Max. Feldlängen)

**Comment by marcusthierfelder** [comment]
Schade das hier kein Statement kommt.

**Comment by mge-medisoftware** [comment]
Zumal das durchaus kein rein "akademisches" Problem ist. Wird Impfnachweis (Erst/Abschluss/Booster..) von unterschiedlichen Institutionen erstellt sind Namensangaben teils unterschiedlich (z.B. mit/ohne Titel) oder schlicht wegen Verwendung eGK-Daten in Praxis und Ausweisdaten in Apotheke als Datengrundlage. 
Die CovPassCheck App lehnt dann scheinbar neuen Impfnachweis ab mit Meldung wie "Zertifikat nicht für den identischen Patienten" o.ä.

**Reply by marcusthierfelder** [reply]
Ja, wir haben uns dazu entschieden Namenszusatz und Vorsatzwort mit in den Nachnamen zu tun und den Titel zu ignorrieren... so wie es die EU verlangt. Also das was auf dem Personalausweis auf der Rückseite steht.
Wir haben aber auch gehört das Apotheken den Titel mit übernehmen

**Comment by jleufgen** [comment]
@marcusthierfelder vielen Dank für deinen Diskussionspunkt! Der Name auf den das Zertifikat ausgestellt werden soll, muss dem Eintrag im Personalausweis entsprechen. Der Titel wird dem Vornamen und der Adel wird dem Nachnamen zugeordnet.

**Reply by marcusthierfelder** [reply]
Vielen Dank.

Kurz zur Klarstellung:
Der Nachname soll so befüllt werden wie auf dem Personalausweisrückseite.
Der Vorname soll **entgegen** dem Personalausweisrückseite zusätzlich zum Vornamen noch den Titel enthalten (Der Titel ist kein Bestandteil des machinen lesbaren Teils) richtig?

Frage aus Interesse: Hätte ich diese Regeln irgendwo finden können?

**Reply by ChristianFischerJena** [reply]
Hallo, auch wir wissen nicht wie der Name übergeben werden soll. Leider ist das ein recht kritischer Punkt. Es gab diesbezüglich auch schon Beschwerden von Patienten, die am Flughafen festsaßen, weil der Name eben nicht exakt überein gestimmt hat.

@jleufgen : Ihre Antwort verwirrt mich etwas. Der Titel ist im maschinenlesbaren Teil (MRZ) des Personalausweises nicht enthalten. Daher bin ich verwirrt, dass sie gleichzeitig schreiben, dass der Titel dem Vornamen zugeordnet werden soll. Wäre es möglich da bitte noch einmal Klarheit zu schaffen?

**Reply by marcusthierfelder** [reply]
@jleufgen: es würde uns freuen wenn wir hier eine Klarstellung (Bestätigung/Verneinung) zur gestellten Frage bekommen könnten.

**Reply by jleufgen** [reply]
@marcusthierfelder der Name muss dem Namen im Ausweis entsprechen.

**Reply by Bergreiter** [reply]
Wie können die Namen auf dem Ausweis entsprechen wenn man nicht den maschinenlesbaren Teil des Namens festlegen kann?

**Reply by GisoSchroederSAP** [reply]
Die ausstellende Partei legt fest, wie die Umsetzung der Namensfelder erfolgt. Grundlage dafür ist Empfehlung aus dem Luftverkehrswesen ICAO 9303. Ob die Umsetzung von Umlaute, Akzenten, middle names usw. manuell oder maschinell erledigt wird, ist dabei unerheblich, es muss am Ende immer dasselbe Ergebnis rauskommen, das auch heute schon auf dem Ausweis und im Pass steht.

**Comment by GisoSchroederSAP** [comment]
Die ICAO Dokumentation schreibt dazu: 
![image](https://user-images.githubusercontent.com/69585203/144451257-462e28c4-fe4f-49ed-a9c5-4bf9d0c34a70.png)

MRZ ist die _machine readable zone_  (also der Teil, der mit standardisierter Schreibweise machinen-lesbar sein soll), im Gegensatz zur VIZ _visible inspection zone_ (mit Bild und alle Sonderzeichen, Titeln usw.)

Das Pass- und Personalausweisgesetz  wurde ja schon zitiert und zieht zur Umseztung des Datensatz des Meldewesens heran, in dem es explizit auch um den Doktorgrad geht (§ 3 Abschnitt 1, Punkt 4):
![image](https://user-images.githubusercontent.com/69585203/144452257-05b8add9-dc9a-40e6-a125-4ad3e62c5930.png)
